CONTRIBUTING.md

```diff
@@ -12,29 +9,104 @@ You can contribute to Turing in several ways:
 - Create articles and documentation for users and contributors
 - Help others answering questions about Turing
 
-### Report bugs
+The following sections provide some suggestions and guidelines you should follow:
+- [Code of Conduct](#coc)
+- [Issues and Bugs](#issue)
+- [Feature Requests](#feature)
+- [Submission Guidelines](#submit)
+- [Coding Rules](#rules)
+
+## <a name="coc"></a> Code of Conduct
+
+Help us keep Turing open and inclusive.
+Please read and follow our [Code of Conduct](https://github.com/caraml-dev/turing/blob/main/CODE_OF_CONDUCT.md).
+
+## <a name="issue"></a> Found a Bug?
+
+If you find a bug in the source code, you can help us by [submitting an issue](#submit-issue) to our [GitHub Repository][github].
+Even better, you can [submit a Pull Request](#submit-pr) with a fix.
+
+
+## <a name="feature"></a> Missing a Feature?
+You can *request* a new feature by [submitting an issue](#submit-issue) to our GitHub Repository.
+If you would like to *implement* a new feature, please consider the size of the change in order to determine the right steps to proceed:
+
+* For a **Major Feature**, first open an issue and outline your proposal so that it can be discussed.
+  This process allows us to better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
```

Comment:
Nit: extra space at the beginning.

Reply:
Oh that's on purpose, might be clearer if you take a look at the 'Preview'